src/app/app.component.ts

```diff
@@ -1,16 +1,34 @@
 import { Component } from '@angular/core';
+import { Platform } from '@ionic/angular';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: 'app.component.html',
   styleUrls: ['app.component.scss'],
 })
 export class AppComponent {
+ 
   public appPages = [
-    { title: 'Created Sessions', url: '', icon: 'person-add' },
-    { title: 'Language', url: '', icon: 'language' },
-    { title: 'Settings', url: '', icon: 'settings' },
-    { title: 'Help', url: '', icon: 'help-circle' }
+    { title: 'LABELS.CREATED_SESSIONS', url: '', icon: 'person-add' },
+    { title: 'LABELS.LANGUAGE', url: '', icon: 'language' },
+    { title: 'LABELS.SETTINGS', url: '', icon: 'settings' },
+    { title: 'LABELS.HELP', url: '', icon: 'help-circle' }
   ];
-  constructor() {}
+  constructor(
+    private translate :TranslateService,
+    private platform : Platform
+  ) {
+    this.initializeApp();
+  }
+
+
+  initializeApp() {
+    this.platform.ready().then(() => {
+      this.languageSetting();
+    });
+  }
+  languageSetting() {
+        this.translate.use('en');
```

Comment:
check for a local storage value and if no values are there then default it to en